examples/book_translation/translate_latex_book.ipynb

```diff
@@ -224,14 +224,14 @@
     "\n",
     "{sample_translation[1]}\n",
     "'''\n",
-    "    response = openai.Completion.create(\n",
+    "    response = openai.chat.completions.create(\n",
```

Comment:
import openai
upgrade to new version and change to cilent variable in the file.